manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
If only one connects, you should still get data. I think polling interval is 10 or 15s.